052-cruise-control-api-users.md

```diff
@@ -0,0 +1,261 @@
+# Add ability to create Cruise Control REST API users
+
+This proposal is about giving developers the ability to create REST API users for the Cruise Control REST API.
+This would allow roles and permissions to be defined to allow developers and third-party applications to access the Cruise Control REST API without having to disable HTTP basic authentication.
+
+## Current situation
+
+At this time, Strimzi creates two Cruise Control API users:
+
+* `admin`      - Used by Strimzi for Cruise Control admin operations.
+* `heathcheck` - Used by Strimzi for checking the readiness of the Cruise Control application.
+
+The access to these API users are limited to Strimzi.
+So when HTTP basic authentication is enabled for Cruise Control, no one can access the the Cruise Control API other than Strimzi.
+
+## Motivation
+
+There are a couple use cases where developers would want to access the Cruise Control API without disabling API security:
+
+* Monitoring a Strimzi-managed Kafka cluster with the Cruise Control UI.
+* Gathering Cruise Control specific statistical information that is not currently available via Strimzi Operator or Cruise Control sensor metrics e.g. detailed information surrounding cluster and partition load and user tasks.
+* Experiment with Cruise Control operations not currently supported by Strimzi.
+* Debugging Cruise Control in a secured environment.
+
+At this time, Strimzi Cruise Control integration offers limited access to Cruise Control data and features.
+Although this limited access protects developers from preforming potentially dangerous cluster operations it also prevents developers from accessing harmless monitoring data.
```

Comment:
`preventing them from dangerous cluster operations` sounds like preventing users from using advanced features. But my understanding was that there are serious concerns that this will interfere with the Strimzi Operators. I think these are two different things where as any conflicts with Strimzi are significantly more serious. So this deserves a clarification.